linuxkit/linuxkit
# tpm tools

**- What I did**
Create an image with the tpm 1.2 software stack (tss), as well as the necessary daemon `tcsd` (configured to start by default).

Also adds an example and a doc.

**- How I did it**
1. `FROM linuxkit/alpine`
2. install necessary packages
3. Download specific version of trousers (tss daemon and lib) and tpm-tools
4. compile and save to `/out/`
5. Load into new `FROM scratch` image

**- How to verify it**
Run it on a laptop/server with tpm 1.2 chip and see that `tcsd` starts and you can communicate with it.

**- Description for the changelog**

Add tpm 1.2 software stack support

**- A picture of a cute animal (not mandatory but encouraged)**
![gremlin](http://actionflickchick.com/superaction/wp-content/uploads/2009/12/gremlins0zf.jpg)

**Note:** testing on this is a bit slow, so don't approve until `[WIP]` removed.
